### ISSUE TITLE
[bug] lodash/prefer-lodash-method false positive?

### ISSUE THREAD

**Issue by wawyed** [wix/eslint-plugin-lodash#119]
I'm currently getting lots of there warnings, but AFAIK I am using the right method.

>  warning  Prefer '_.forEach' over the native function                 lodash/prefer-lodash-method

```javascript
 _.forEach(methodList, function(method) {
  console.log(method);
});
```

This is my rule config:

```json
    "lodash/prefer-lodash-method": [
      1
    ],
```

Eslint version: "3.9.1"
Eslint lodash plugin version: "2.1.7"

**Comment by ganimomer** [comment]
How are you loading `_`? Is it a global or is it imported earlier in the file?
If it's not imported via commonjs `require` or via ES6 `import`, (e.g. if it's via a script tag) then you need to add a `pragma` setting to your config. You can extend the `plugin:lodash/canonical` config, or add this to your `.eslintrc` file:
```json
{
  "settings": {
    "lodash": {
        "pragma": "_"
      }
  }
}
```

**Comment by wawyed** [comment]
Thank you! That fixed the issue. :)
